src/components/FoodList.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Foodbox from './FoodBox'
+
+const FoodList = (props) => {
+    return (
+        <div className="column is-two-thirds">
+            {
+                props.foods.map((item, i) => {
+                    return <Foodbox 
```

Comment:
if you are adding a JSX element in multiple lines it is better to start below the return, not in the same line. and  wrap it inside ( ).
return (
    <Foodbox 
    item={item} 
    key={i} 
    id={i} 
    onAdd={props.onAdd} 
    />
)

or like this:

        return <Foodbox item={item} key={i} id={i} onAdd={props.onAdd} />;